Support for nestjs v10?

Current version of nestjs-otel does not support nestjs v10:

> npm ERR! Found: @nestjs/common@10.0.1
> npm ERR! node_modules/@nestjs/common
> npm ERR!   @nestjs/common@"10.0.1" from the root project
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer @nestjs/common@"^9.0.0" from nestjs-otel@5.1.2
> npm ERR! node_modules/nestjs-otel
> npm ERR!   nestjs-otel@"5.1.2" from the root project

peerDependencis defined at package.json:
```json
  "peerDependencies": {
    "@nestjs/common": "^9.0.0",
    "@nestjs/core": "^9.0.0"
  },
```

Are there any plans for supporting nestjs v10? Installing packages with `--force` seems to work, we are not seeing any issues..

--- a/package.json
+++ b/package.json
@@ -72,8 +72,8 @@
     "typescript": "5.0.4"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.0.0",
-    "@nestjs/core": "^9.0.0"
+    "@nestjs/common": "^9.0.0 || ^10.0.0",
+    "@nestjs/core": "^9.0.0 || ^10.0.0"
   },
   "jest": {
     "moduleFileExtensions": [

--- a/package.json
+++ b/package.json
@@ -43,11 +43,11 @@
   "devDependencies": {
     "@commitlint/cli": "^17.6.1",
     "@commitlint/config-conventional": "^17.6.1",
-    "@nestjs/common": "^9.4.0",
-    "@nestjs/core": "^9.4.0",
-    "@nestjs/platform-express": "^9.4.0",
-    "@nestjs/platform-fastify": "^9.4.0",
-    "@nestjs/testing": "^9.4.0",
+    "@nestjs/common": "^10.0.3",
+    "@nestjs/core": "^10.0.3",
+    "@nestjs/platform-express": "^10.0.3",
+    "@nestjs/platform-fastify": "^10.0.3",
+    "@nestjs/testing": "^10.0.3",
     "@opentelemetry/exporter-prometheus": "^0.38.0",
     "@opentelemetry/sdk-metrics": "^1.12.0",
     "@opentelemetry/sdk-trace-node": "^1.12.0",
@@ -72,8 +72,8 @@
     "typescript": "5.0.4"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.0.0 || ^10.0.0",
-    "@nestjs/core": "^9.0.0 || ^10.0.0"
+    "@nestjs/common": "^10.0.0",
+    "@nestjs/core": "^10.0.0"
   },
   "jest": {
     "moduleFileExtensions": [

